services/traction/api/endpoints/routes/v1/tenant/issuer/credentials.py

```diff
@@ -0,0 +1,130 @@
+import logging
+
+from fastapi import APIRouter, Depends
+from sqlalchemy.ext.asyncio import AsyncSession
+from starlette import status
+
+from api.services.v1 import issuer_service
+from api.db.models.issue_credential import IssueCredentialRead
+
+
+from api.db.models.v1.contact import Contact
+
+from api.endpoints.dependencies.tenant_security import get_from_context
+from api.endpoints.dependencies.db import get_db
+
+
+from api.endpoints.models.tenant_workflow import (
+    TenantWorkflowStateType,
+)
+
+from api.endpoints.models.v1.issuer import (
+    CredentialsListResponse,
+    CredentialItem,
+    IssueCredentialPayload,
+    RevokeSchemaPayload,
+)
+
+router = APIRouter()
+logger = logging.getLogger(__name__)
+
+
+@router.get("/", response_model=CredentialsListResponse)
+async def get_issued_credentials(
+    state: TenantWorkflowStateType | None = None,
+    cred_issue_id: str | None = None,
+    db: AsyncSession = Depends(get_db),
+) -> CredentialsListResponse:
+    # this should take some query params, sorting and paging params...
+    wallet_id = get_from_context("TENANT_WALLET_ID")
+    tenant_id = get_from_context("TENANT_ID")
+
+    data = await issuer_service.get_issued_credentials(
+        db, tenant_id, wallet_id, None, cred_issue_id, state
+    )
+    resp_data = [
+        CredentialItem(
+            **d.__dict__,
+            contact_id=d.credential.contact_id,
+            credential_id=d.credential.id,
+            status="v0",  # v0
```

Comment:
We know there will be a `status` member to this model, we just don't know what it will be yet. so i thought this was a reasonable placeholder. 

Unless we want to discuss what it means? or is there going to be a global enum for 'ItemStatus' that we want to use.